charts/argo/templates/workflow-controller-config-map.yaml

```diff
@@ -67,3 +67,7 @@ data:
     {{- if .Values.controller.workflowDefaults }}
     workflowDefaults:
 {{ toYaml .Values.controller.workflowDefaults | indent 6 }}{{- end }}
+    {{- with .Values.server.sso }}
+    server:
```

Comment:
Looking at the docs, `sso` does not sit under `server` and sits directly under `data` https://github.com/argoproj/argo/blob/2dae724496a96ce2e0993daea0a3b6a473f784da/docs/workflow-controller-configmap.yaml

Reply:
Ah, good good point - thanks for the catch! Updated.